discovery/client.go

```diff
@@ -68,6 +68,8 @@ func (clnt *Client) AddBackend(bk Backend) {
 	clnt.lock.Lock()
 	defer clnt.lock.Unlock()
 
+
```

Comment:
These two blank lines seems to be left here occasionally, as result of editing. Please, remove